spec/models/league_spec.rb

```diff
@@ -21,14 +21,27 @@
 
 require 'rails_helper'
 
-describe "League model", type: :model do
+describe 'League model', type: :model do
 
-  it "is valid when produced by a factory" do
+  let(:league) { FactoryBot.build(:league) }
+  it 'is valid when produced by a factory' do
     league = FactoryBot.build(:league)
     expect(league).to be_valid
   end
 
-  describe "Generating league schedule with default values" do
+  it 'should not validate without gameday duration' do
+    league = FactoryBot.build(:league, gameday_duration: nil)
```

Comment:
Is this any different than saying `league.gameday_duration = nil`? Since you create a league object already in line 26

Reply:
no it's not I'll change so it's easier to read